iambic/plugins/v0_1_0/aws/cloud_formation/templates/IambicSpokeRole.yml

```diff
@@ -30,6 +30,13 @@ Resources:
           PolicyDocument:
             Version: '2012-10-17'
             Statement:
+              - Effect: Allow
```

Comment:
this is already covered in Sid `SqsWriteAccessForResourceMonitoring `

Reply:
Good catch, fixed!